### ISSUE TITLE
[BUG]: Error "Field Book isn't responding" upon scanning qr code with the scanner in the Collect Toolbar

### ISSUE THREAD

**Issue by adtbuild** [PhenoApps/Field-Book#1128]
### Description

When I click the scanner icon (specifically the one located in the collect toolbar) and focus the scanner on a QR code, the screen freezes with an image of the QR code, then the screen goes black, then I get the error from my phone that Field Book is not responding and I either can wait or close the app. However, the magnifying glass icon located on the homepage in the bottom right corner does work properly to scan my QR codes without the app crashing. This is a consisent error accross three different devices.

### Field Book version

6.0.2

**Comment by trife** [comment]
@adtbuild Do you have the experimental MLKit enabled? Can you upload a pic of one of your QR codes? I'm not seeing any crashes in 6.0.2 when I scan QR/barcodes.

**Comment by adtbuild** [comment]
@trife I did not previously have the MLKit enabled, but upon enabling it just now, it was faster and worked well on one of my phones without crashing. However, this phone is version 5.6.25 and was just recently cleared of its data. Upon enabling it on two other phones, the app still crashed. One was 6.0.2 and the other was 6.0.3. To note, these two phones have about 500mb data already collected/stored on them.

**Comment by trife** [comment]
@adtbuild I'm not seeing this behavior so wondering if it's related to your QR code. Can you upload a picture of one of your QR codes?

**Comment by adtbuild** [comment]
Yes, attached is a pdf of our qr codes. [Avery94272_2024_Peduncle_Geneva_repository_all_accessions_fieldbook_images_LABELS_FINAL_pg1.pdf](https://github.com/user-attachments/files/18274833/Avery94272_2024_Peduncle_Geneva_repository_all_accessions_fieldbook_images_LABELS_FINAL_pg1.pdf)

**Comment by trife** [comment]
@adtbuild I'm unable to replicate your crash.

@chaneylc Any ideas?

**Comment by chaneylc** [comment]
Apologies for the late reply, can you send us your app database so we can inspect this further?

**Comment by bshallwachs** [comment]
I was having this issue too. I went ahead and went to settings>features>move to unique ID> "button uses direct camera scan"

Prior to that it was working, then the scanner magically quit working, so now its working after applying this feature (hope it continues to work)

**Comment by trife** [comment]
@bshallwachs That's likely due to some settings changes that we made with v6 but should be fixed now. Sorry for the confusion.

Closing this since we haven't received the database for further testing.
